sklearn/feature_extraction/text.py

```diff
@@ -1322,17 +1333,17 @@ class TfidfVectorizer(CountVectorizer):
         Otherwise the input is expected to be the sequence strings or
         bytes items are expected to be analyzed directly.
 
-    encoding : string, 'utf-8' by default.
+    encoding : string, 'utf-8' (default='utf-8')
```

Comment:
This is no longer consistent with CountVectorizer. The point of this PR is to make TfidfVectorizer's parameter descriptions identical to those in either CountVectorizer or TfidfTransformer because TfidfVectorizer is a composition of those two.